cabal.project

```diff
@@ -5,8 +5,6 @@ packages: ./
           ./haddock-library
           ./haddock-test
 
-with-compiler: ghc-9.4
```

Comment:
oops, probably my fault from a bad merge resolve. `with-compiler` is specified above